saleor/core/utils/editorjs.py

```diff
@@ -69,27 +69,27 @@ def clean_list_item(blocks, block, plain_text_list, to_string, index):
     for item_index, item in enumerate(block["data"]["items"]):
         if not item:
             return
-        new_text = clean_text_data(item)
         if to_string:
-            plain_text_list.append(strip_tags(new_text))
+            plain_text_list.append(strip_tags(item))
         else:
+            new_text = clean_text_data_block(item)
```

Comment:
`clean_text_data_block` makes regex operation inside to fetch <a href> links. It is not needed when we want to convert the data into plain text, as `strip_tags` drops all tags.